panic: runtime error: invalid memory address or nil pointer dereference while fetching

```
go build ; and ./goval-dictionary fetch-redhat -debug 7                                                                                                                                                                                                                                                                                                       INFO[07-03|10:31:56] Fetching...                              URL=https://www.redhat.com/security/data/oval/com.redhat.rhsa-RHEL7.xml.bz2
INFO[07-03|10:31:57] Finished to fetch OVAL definitions.
INFO[07-03|10:31:58] Fetched                                  URL=https://www.redhat.com/security/data/oval/com.redhat.rhsa-RHEL7.xml.bz2 OVAL definitions=659
INFO[07-03|10:31:58] Skip (Same Timestamp)                    Family=redhat Version=7
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x40 pc=0x423ba6d]

goroutine 1 [running]:
database/sql.(*Tx).rollback(0x0, 0x4672f00, 0xc420c17801, 0x5f799a0)
        /Users/kanbe/.goenv/versions/1.10.1/src/database/sql/sql.go:1944 +0x2d
database/sql.(*Tx).Rollback(0x0, 0x5f79908, 0x0)
        /Users/kanbe/.goenv/versions/1.10.1/src/database/sql/sql.go:1963 +0x30
github.com/kotakanbe/goval-dictionary/vendor/github.com/jinzhu/gorm.(*DB).Rollback(0xc420867cb0, 0x45bf840)
        /Users/kanbe/go/src/github.com/kotakanbe/goval-dictionary/vendor/github.com/jinzhu/gorm/main.go:505 +0xa4
github.com/kotakanbe/goval-dictionary/db/rdb.(*Driver).InsertFetchMeta(0xc42034e1b0, 0x0, 0xc420022fca, 0x1d, 0x0, 0xed2ca4b32, 0x0, 0x0, 0x0)
        /Users/kanbe/go/src/github.com/kotakanbe/goval-dictionary/db/rdb/rdb.go:225 +0x24f
github.com/kotakanbe/goval-dictionary/commands.(*FetchRedHatCmd).Execute(0xc420298a00, 0x4705400, 0xc420024098, 0xc420288cc0, 0x0, 0x0, 0x0, 0x0)
        /Users/kanbe/go/src/github.com/kotakanbe/goval-dictionary/commands/fetch-redhat.go:166 +0xc33
github.com/kotakanbe/goval-dictionary/vendor/github.com/google/subcommands.(*Commander).Execute(0xc4200b2000, 0x4705400, 0xc420024098, 0x0, 0x0, 0x0, 0x0)
        /Users/kanbe/go/src/github.com/kotakanbe/goval-dictionary/vendor/github.com/google/subcommands/subcommands.go:141 +0x29f
github.com/kotakanbe/goval-dictionary/vendor/github.com/google/subcommands.Execute(0x4705400, 0xc420024098, 0x0, 0x0, 0x0, 0xc4202b65a8)
        /Users/kanbe/go/src/github.com/kotakanbe/goval-dictionary/vendor/github.com/google/subcommands/subcommands.go:371 +0x5f
main.main()
        /Users/kanbe/go/src/github.com/kotakanbe/goval-dictionary/main.go:52 +0x348
```

--- a/db/rdb/debian.go
+++ b/db/rdb/debian.go
@@ -34,7 +34,7 @@ func (o *Debian) InsertOval(root *models.Root, meta models.FetchMeta, driver *go
 	r := tx.Where(&models.FetchMeta{FileName: meta.FileName}).First(&oldmeta)
 	if !r.RecordNotFound() && oldmeta.Timestamp.Equal(meta.Timestamp) {
 		log15.Info("Skip (Same Timestamp)", "Family", root.Family, "Version", root.OSVersion)
-		return nil
+		return tx.Rollback().Error
 	}
 	log15.Info("Refreshing...", "Family", root.Family, "Version", root.OSVersion)
 

--- a/db/rdb/oracle.go
+++ b/db/rdb/oracle.go
@@ -34,7 +34,7 @@ func (o *Oracle) InsertOval(root *models.Root, meta models.FetchMeta, driver *go
 	r := tx.Where(&models.FetchMeta{FileName: meta.FileName}).First(&oldmeta)
 	if !r.RecordNotFound() && oldmeta.Timestamp.Equal(meta.Timestamp) {
 		log15.Info("Skip (Same Timestamp)", "Family", root.Family, "Version", root.OSVersion)
-		return nil
+		return tx.Rollback().Error
 	}
 	log15.Info("Refreshing...", "Family", root.Family, "Version", root.OSVersion)
 

--- a/db/rdb/rdb.go
+++ b/db/rdb/rdb.go
@@ -209,7 +209,7 @@ func (d *Driver) InsertFetchMeta(meta models.FetchMeta) error {
 	oldmeta := models.FetchMeta{}
 	r := tx.Where(&models.FetchMeta{FileName: meta.FileName}).First(&oldmeta)
 	if !r.RecordNotFound() && oldmeta.Timestamp.Equal(meta.Timestamp) {
-		return nil
+		return tx.Rollback().Error
 	}
 
 	if r.RecordNotFound() {

--- a/db/rdb/redhat.go
+++ b/db/rdb/redhat.go
@@ -34,7 +34,7 @@ func (o *RedHat) InsertOval(root *models.Root, meta models.FetchMeta, driver *go
 	r := tx.Where(&models.FetchMeta{FileName: meta.FileName}).First(&oldmeta)
 	if !r.RecordNotFound() && oldmeta.Timestamp.Equal(meta.Timestamp) {
 		log15.Info("Skip (Same Timestamp)", "Family", root.Family, "Version", root.OSVersion)
-		return nil
+		return tx.Rollback().Error
 	}
 	log15.Info("Refreshing...", "Family", root.Family, "Version", root.OSVersion)
 

--- a/db/rdb/suse.go
+++ b/db/rdb/suse.go
@@ -33,7 +33,7 @@ func (o *SUSE) InsertOval(root *models.Root, meta models.FetchMeta, driver *gorm
 	r := tx.Where(&models.FetchMeta{FileName: meta.FileName}).First(&oldmeta)
 	if !r.RecordNotFound() && oldmeta.Timestamp.Equal(meta.Timestamp) {
 		log15.Info("Skip (Same Timestamp)", "Family", root.Family, "Version", root.OSVersion)
-		return nil
+		return tx.Rollback().Error
 	}
 	log15.Info("  Refreshing...", "Family", root.Family, "Version", root.OSVersion)
 

--- a/db/rdb/ubuntu.go
+++ b/db/rdb/ubuntu.go
@@ -34,7 +34,7 @@ func (o *Ubuntu) InsertOval(root *models.Root, meta models.FetchMeta, driver *go
 	r := tx.Where(&models.FetchMeta{FileName: meta.FileName}).First(&oldmeta)
 	if !r.RecordNotFound() && oldmeta.Timestamp.Equal(meta.Timestamp) {
 		log15.Info("Skip (Same Timestamp)", "Family", root.Family, "Version", root.OSVersion)
-		return nil
+		return tx.Rollback().Error
 	}
 	log15.Info("Refreshing...", "Family", root.Family, "Version", root.OSVersion)
 

